src/crush/CrushCompiler.cc

```diff
@@ -815,6 +815,44 @@ void CrushCompiler::dump(iter_t const& i, int ind)
     dump(i->children.begin() + j, ind+1); 
 }
 
+int CrushCompiler::adjust_bucket_item_place(iter_t const &i)
```

Comment:
could you elaborate a little bit what this function is doing in the comments? personally, i think "adjust" is quite vague.